j2ee.persistence/src/org/netbeans/modules/j2ee/persistence/ui/resources/persistence-1.0.xml

```diff
@@ -1,5 +1,24 @@
 <?xml version="1.0" encoding="UTF-8"?>
+<!--
 
+    Licensed to the Apache Software Foundation (ASF) under one
+    or more contributor license agreements.  See the NOTICE file
+    distributed with this work for additional information
+    regarding copyright ownership.  The ASF licenses this file
+    to you under the Apache License, Version 2.0 (the
+    "License"); you may not use this file except in compliance
+    with the License.  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing,
+    software distributed under the License is distributed on an
+    "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+    KIND, either express or implied.  See the License for the
+    specific language governing permissions and limitations
+    under the License.
+
+-->
```

Comment:
persistance-*.xml files are copied:
https://github.com/apache/incubator-netbeans/blob/ff1bc6c47951f9f42231c0c06ff58fbfc99f810e/j2ee.persistence/src/org/netbeans/modules/j2ee/persistence/provider/ProviderUtil.java#L866-L872

So, please undo changes then, should be added to rat excludes.